radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -172,23 +172,22 @@ impl FeeTable {
     #[inline]
     pub fn before_invoke_cost(&self, _actor: &Actor, input_size: usize) -> u32 {
         add(
-            13 / CPU_INSTRUCTIONS_TO_COST_UNIT,
+            10 / CPU_INSTRUCTIONS_TO_COST_UNIT,
```

Comment:
This equals 0, should we just change this to 1?

Reply:
Which function is the `10` derived from?